package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Add sourcemaps to vinyl files, extracted from gulp-sourcemaps by Florian Reiterer (https://github.com/floridoo/gulp-sourcemaps)",
   "main": "index.js",
   "scripts": {
-    "test": "faucet test/*.js",
-    "tap": "tape test/*.js"
+    "test": "mocha test/*.js",
```

Comment:
use `mocha test/` instead of `*.js` because Windows doesn't work well with globs on the command line